packages/blip/src/platform/windows.adapter.ts

```diff
@@ -1,6 +1,130 @@
 import { BlipAdapter } from "./blip.adapter";
 
+const { exec } = require("child_process");
+
+/**
+ * A blip adapter used on Windows Operating systems.
+ * 
+ * @param {BlipAdapterEmitter}
+ * @param {BlipAdapterOpts}
+ */
 const WindowsAdapter: BlipAdapter = async (emitters, opts) => {
-    console.log('Windows Adapter works!');
+    let running = true;
+    let occupied = false;
+
+    // Array that contains the different locales for the active status
+    const ACTIVE_STATUS_LOCALES = ["aktiv", "active"];
+    const INTERVAL_MS = opts.interval * 1000;
+
+ 	/**
+ 	 * Updates the occupancy if it changed.
+ 	 */
+    const updateOccupancy = async () => {
+        // trigger next cycle when still running
+        if (running) setTimeout(updateOccupancy, INTERVAL_MS);
+
+        // Execute command to retrieve session list
+        let cmd = exec("query session", "", processOutputData);    
+    }
+
+	/**
+	 * Callback handler for the "query session" execution.
+	 *
+	 * Processes the output and updates the occupied flag according to the output data.
+	 * 
+	 * @param {any} error Represents an execution error if one occured.
+	 * @param {string} stdout Output stream of the executed command.
+	 * @param {string} stderr Error stream of the executed command.
+	 */
+	const processOutputData = async (error: any, stdout: string, stderr: string) => {
+   		
+   		if(error.code != 0 && error.code != 1) {
+   			console.error(`Error: ${error}`);
```

Comment:
If `error` is an object it will simply log: `Error: [object Object]`. You probably want this:
```suggestion
   			console.error(`Error: ${JSON.stringify(error)}`);
```